### PR TITLE
Update changelog generation

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Updated the changelog system

image

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
This also fixes #17730

**Comment by FluffyGhoster** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

File: tgui/packages/tgui/interfaces/Changelog.js [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+              return this.getData(dates[index]);
+            }}
+          />
+        </Stack.Item>
+      </Stack>
+    );
+
+    const header = (
+      <Section>
+        <h1>Traditional Games Space Station 13</h1>
```
Comment (Arrow768) [inline tgui/packages/tgui/interfaces/Changelog.js:180]: This is Aurora Station.
Not TG Station

File: tgui/packages/tgui/interfaces/Changelog.js [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+        <p>
+          <b>Thanks to: </b>
+          Baystation 12, /vg/station, NTstation, CDK Station devs,
```
Comment (Arrow768) [inline tgui/packages/tgui/interfaces/Changelog.js:183]: Tgstation, baystation should be added

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**